PatternPal/PatternPal.Extension/ViewModels/PatternResultViewModel.cs

```diff
@@ -98,6 +97,7 @@ public SolidColorBrush GetProgressBarColor()
         }
     }
 
+    // TODO Unused
```

Comment:
@reviewer

Reply:
Didn't do since was a bit outside my knowledge, so documentation would be prefereable from @DanielvanDamme or @KnapSac